kombu/transport/redis_cluster.py

```diff
@@ -364,9 +366,10 @@ def _poll_error(self, cmd, conn, **options):
             resp = self.parse_response(conn, 'BRPOP', **options)
             if resp:
                 self.deliver_response(resp)
-        except self.connection_errors as e:
+        except Exception:
```

Comment:
if we catch all Exception and do nothing, what's the point to use try-catch block in the caller of `parse_response` rather than make `parse_response` handle all errors

Reply:
We need to handle differently on error of `parse_response` for `_brpop_read` and `_poll_error`

Reply:
you mean throwing `Empty()`?

Reply:
yes!